catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/NsiliEndpoint.java

```diff
@@ -46,7 +46,7 @@
 
      public static final int DEFAULT_MAX_NUM_RESULTS = 500;
 
-    public static final int DEFAULT_CORBA_PORT = 20003;
+    public static final int DEFAULT_CORBA_PORT = 2809;
```

Comment:
Default ORB port 20003 changed to deconflict with the port allocations done by the integration tests (see AbstractIntegrationTest). This conflict caused an issue where the NSILI endpoint could not initialize when running itests. We investigated a standard port for IIOP and did not find one universally used.